### ISSUE TITLE
ip -s link fail some times

### ISSUE THREAD

**Issue by Unitech** [keymetrics/pm2-server-monit#39]
https://github.com/pm2-hive/pm2-server-monit/blob/fc31bcc9dc2dda4b6dba1b0287c3fa9fb53f0c8f/lib/netstat.js#L9
(+return on error?)

```bash
root@ubuntu-xenial: ~
>>> time ip -s link
163.172.175.62

real	0m0.174s
user	0m0.000s
sys	0m0.000s

root@ubuntu-xenial: ~
>>> time ip -s link
^C

real	1m22.835s
user	0m0.000s
sys	0m0.000s

```

**Comment by btok** [comment]
+1 
Anyone else having this issue ? Happening lately a lot.


0|pm2-serv | exec: internal error
0|pm2-serv | exec: internal error
0|pm2-serv | exec: internal error
0|pm2-serv | exec: internal error
0|pm2-serv | Error: ip -s a failed to execute
0|pm2-serv |     at /home/ubuntu/.pm2/node_modules/pm2-server-monit/lib/netstat.js:11:13
0|pm2-serv |     at /home/ubuntu/.pm2/node_modules/shelljs/src/exec.js:180:7
0|pm2-serv |     at exithandler (child_process.js:213:5)
0|pm2-serv |     at ChildProcess.errorhandler (child_process.js:225:5)
0|pm2-serv |     at emitOne (events.js:96:13)
0|pm2-serv |     at ChildProcess.emit (events.js:188:7)
0|pm2-serv |     at Process.ChildProcess._handle.onexit (internal/child_process.js:213:12)
0|pm2-serv |     at onErrorNT (internal/child_process.js:359:16)
0|pm2-serv |     at _combinedTickCallback (internal/process/next_tick.js:74:11)
0|pm2-serv |     at process._tickDomainCallback (internal/process/next_tick.js:122:9)

**Comment by vmarchaud** [comment]
Fixed in 3.0.0
